tools/exporters/data_dictionary_builder.php

```diff
@@ -185,7 +195,7 @@
                     "Type"        => $bits[0],
                     "Description" => $bits[2],
                     "SourceField" => $bits[1],
-                    "SourceFrom"  => $table,
+                    "SourceFrom"  => $testname,
```

Comment:
I don't think this part is accurate. SourceFrom is supposed to be the table in the quat tables that the data is sourced from.. but I also don't think it matters, because the quat tables don't exist/aren't used anymore and as far as I know this isn't used anywhere, so I'll just merge it..